src/Symfony/Component/Form/Form.php

```diff
@@ -89,7 +89,7 @@ class Form implements \IteratorAggregate, FormInterface
     private $submitted = false;
 
     /**
-     * @var ClickableInterface|null The button that was used to submit the form
+     * @var Button|ClickableInterface|null The button that was used to submit the form
```

Comment:
Should be `FormInterface|ClickableInterface|null` I guess.

Reply:
Thanks @xabbuh, I updated to `FormInterface`.